charts/vector/templates/daemonset.yaml

```
@@ -12,7 +12,7 @@ spec:
     matchLabels:
       {{- include "vector.selectorLabels" . | nindent 6 }}
   {{- with .Values.updateStrategy }}
```

```suggestion
   minReadySeconds: {{ $.Values.minReadySeconds }}
   {{- with .Values.updateStrategy }}
```

@rbakhtaraev would you mind swapping these lines in this proposal?

---

If moved outside of the `with` block, the `$` should be dropped as well, I beleive.

---

https://github.com/vectordotdev/helm-charts/pull/308 This PR already has the correct changes.

---

This is true. @spencergilbert how do you feel about reviewing the other PR instead? Or would you prefer moving forward on this proposal?

---

yep, fixed it

---

awesome, thanks.